src/fragments/lib/geo/js/google-migration.mdx

```diff
@@ -228,3 +228,133 @@ marker.addListener('click', () => {
+
+<Callout>
+
+This example uses the [MapLibre's geocoder component](https://github.com/maplibre/maplibre-gl-geocoder) to create a search component. To see more options for our `createAmplifyGeocoder` utility function check out the docs [here](https://github.com/aws-amplify/maplibre-gl-js-amplify/blob/main/API.md#createAmplifyGeocoder).
+
+</Callout>
+
+  </Block>
+
+  <Block name="Google Maps">
+
+Using the Google Places JS API you would add a search bar as shown below.
+
+<Callout>
+
+Some lines omitted for brevity, see the Google Maps Platform Places Search Box example for the full application
+
+</Callout>
+
+```js
+// Create the search box and link it to the UI element.
+const input = document.getElementById("pac-input") as HTMLInputElement;
+const searchBox = new google.maps.places.SearchBox(input);
+
+map.controls[google.maps.ControlPosition.TOP_LEFT].push(input);
+
+// Bias the SearchBox results towards current map's viewport.
+map.addListener("bounds_changed", () => {
+  searchBox.setBounds(map.getBounds() as google.maps.LatLngBounds);
+});
+
+// Listen for the event fired when the user selects a prediction and retrieve more details for that place.
+searchBox.addListener("places_changed", () => {
+  const places = searchBox.getPlaces();
+
+  if (places.length == 0) {
+    return;
+  }
+
+  // For each place, get the icon, name and location.
+  places.forEach((place) => {
+    // Create markers for each place
+    // Extend map view for each place
+  });
+  map.fitBounds(bounds);
+});
+```
+
+  </Block>
+
+</BlockSwitcher>
+
+![A search box](/images/geocoder-custom-images.png)
+
+## Add a stand alone search component
+
+Now we can try adding a search bar without adding it to a map which can return results that you can use.
+
+<BlockSwitcher>
+
+  <Block name="Amplify">
+
+With Amplify Geo and MapLibre you can do the following.
+
+```js
+// Create a div to hold the search component
+const el = document.createElement("div");
+el.setAttribute("id", "search");
+document.body.appendChild(el);
+
+// Create the geocoder component and append it to the div you created earlier
+const geocoder = createAmplifyGeocoder();
+document.getElementById("search").appendChild(geocoder.onAdd());
+```
+
+Save your changes and refresh your page and now when you should see a maplibre-gl-geocoder control in the div you created.
+
+<Callout>
+
+This example uses the [MapLibre's geocoder component](https://github.com/maplibre/maplibre-gl-geocoder) to create a search component. To see more options for our `createAmplifyGeocoder` utility function check out the docs [here](https://github.com/aws-amplify/maplibre-gl-js-amplify/blob/main/API.md#createAmplifyGeocoder).
+
+</Callout>
+
+  </Block>
+
+  <Block name="Google Maps">
+
+Using the Google Places JS API you would add a stand alone search bar as shown below.
+
+<Callout>
+
+Some lines omitted for brevity, see the Google Maps Platform Places Search Box example for the full application
```

Comment:
why is this callout needed?

Reply:
Hm because the actual code snippet from GCP is quite long and I didn't want to include the whole thing (https://developers.google.com/maps/documentation/javascript/examples/places-searchbox) but at the same time I didn't want to misrepresent this as being the full example that you could drop into your application

Reply:
Fine with removing it though :)

Reply:
@abdallahshaban557 @thaddmt - Wanted to check on the decision on this callout. Thanks!

Reply:
@abdallahshaban557 @hdworld11 any opinions on this?

Reply:
I think we should just remove it! Will leave it up to @hdworld11 if she thinks otherwise?

Reply:
Removed!

Reply:
Discussed with @thaddmt, I think the data we have here is sufficient to guide the developer on how they can migrant their current implementation to Amplify Geo. So I am okay with removing the callout.